File: Menus/app/src/main/java/com/example/menus/MainActivity.java
package com.example.menus;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.Menu;
import android.view.MenuInflater;
import android.view.MenuItem;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        getMenuInflater().inflate(R.menu.menu_principal, menu);
        return super.onCreateOptionsMenu(menu);
    }

    @Override
    public boolean onOptionsItemSelected(@NonNull MenuItem item) {
        switch (item.getItemId()){
            case R.id.itemSalvar:
                Toast.makeText(MainActivity.this, "ItemSalvar", Toast.LENGTH_LONG).show();
                break;
            case R.id.itemEditar:
                Toast.makeText(MainActivity.this, "ItemEditar", Toast.LENGTH_LONG).show();
                break;
        }
        return super.onOptionsItemSelected(item);
    }
}

File: DialogAccessibility/app/src/main/java/com/example/dialogaccessibility/MainActivity.kt
package com.example.dialogaccessibility

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast
import androidx.appcompat.app.AlertDialog
import com.example.dialogaccessibility.databinding.ActivityMainBinding
import com.example.dialogaccessibility.databinding.DialogBinding

class MainActivity : AppCompatActivity() {

    private lateinit var  binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.btn.setOnClickListener{
            teste()
        }

    }

    private fun teste() {
        val dialog = CustomDialog()
        dialog.show(supportFragmentManager,"tomanocu")
//        var  dialogBinding = DialogBinding.inflate(layoutInflater)
//        val bulder = AlertDialog.Builder(applicationContext).apply {
//            setContentView(dialogBinding.root)
//            setCancelable(true)
//        }.create()
//        dialogBinding.btnPositive.setOnClickListener {
//            Toast.makeText(applicationContext,"botao positivo", Toast.LENGTH_LONG).show()
//        }
//        dialogBinding.btnNegative.setOnClickListener {
//            Toast.makeText(applicationContext,"potao negativo", Toast.LENGTH_LONG).show()
//        }
//        bulder.show()
    }
}